main.cpp

```diff
@@ -56,86 +64,172 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
-    
-    
+    int secondNumber = 3;
+    int thirdNumber = 4;
+    char firstInitial = 66;
+    char lastInitial = 66;
+    char middleInitial = 66;
```

Comment:
`char` types are single alpha-numeric symbols wrapped in single quotes.   you are initializing a `char` with an integer.
- [ ] revise.

Reply:
When would you use the char number? For example 66 is "B"?

Reply:
![image](https://user-images.githubusercontent.com/60301747/80805601-7bf8c180-8b86-11ea-8c98-6567f96d48a2.png)

This must be for something completely different 😀

Reply:
But this is what my "dec" number was referring to.